app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -0,0 +1,83 @@
+module Katello
+  class Api::V2::HostSubscriptionsController < Katello::Api::V2::ApiController
+    before_filter :find_host
+    before_filter :check_subscriptions, :only => [:add_subscriptions, :remove_subscriptions]
+
+    api :GET, "/hosts/:host_id/subscriptions", N_("List a host's subscriptions")
+    param :host_id, Integer, :desc => N_("Id of the host"), :required => true
+    def index
```

Comment:
So we used to do this and IMO it made this controller even more confusing.  You have to fetch different types of objects depending on if its a System or an Activation key and then litter the subscription rabl with quantity information even though a subscription doesn't actually have quantity.    

And the truth is that a list of attached subscriptions to object X is a different 'object' than a subscription.  Candlepin calls that an entitlement.  Now if we wanted to just purely return subscriptions without the quantity I'm perfectly okay with that, but from past experience trying to do it here was too confusing, must simpler to do it somewhere that is focused on hosts/systems.